### ISSUE TITLE
[BUG] When the gltFast project hits its Git LFS limit, the MRTK3 sample project fails to work

### ISSUE THREAD

**Issue by ms-RistoRK** [MixedRealityToolkit/MixedRealityToolkit-Unity#630]
## Describe the bug

A fresh clone of MRTK3 main branch repo fails to load in Unity Editor with Unity Package Manager Error shown next:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/1f70ac85-b166-4b62-a3e4-005a72618d09)

## To reproduce

1. Clone MRTK3 main branch locally
2. Use Unity Hub to add project located in <REPO_ROOT>\UnityProjects\MRTKDevTemplate
3. See error

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/45b0c5d9-d6c5-453c-bb8e-68981de1c362)

## Expected behavior

After Step 2, Unity Editor should open, import packages, without displaying any Unity Package Manager Error.

## Screenshots

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/d541d833-814b-4703-b0eb-111529ba2b45)

## Your setup (please complete the following information)

- Unity Version 2021.3.21f1
- MRTK Version: 3 or Commit [e.g. v3.0] [e.g. [3aa92c5](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/commit/3aa92c50c8db736a895c183c244720155274c5e3)]

## Target platform (please complete the following information)

- Windows 11 Desktop

## Additional context

It seems that a package from https://github.com/atteneder/glTFast is registered in the package manager.

**Comment by AMollis** [comment]
This in some way is an external issue.   We can look at updating the sample to Unity gltf plugin instead, which wouldn't have the same issues.

**Comment by ms-RistoRK** [comment]
Won't fix because:
* We lack the resources to exhaustively test the proposed fix in all devices where MRTK3 is deployed and Qualcomm has suggested to test the proposed fix on their devices.
* The issue is known and accepted.
For historical reference:  The PR that was proposing a solution is this one: https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/631

**Comment by keveleigh** [comment]
Re-opening to continue tracking. Assigning to myself.
